src/assets/i18n/kk.json5

```diff
@@ -1182,28 +1113,22 @@
   "browse.startsWith.months.september": "Қыркүйек",
 
   // "browse.startsWith.submit": "Browse",
-  // TODO Source message changed - Revise the translation
-  "browse.startsWith.submit": "Go",
+  "browse.startsWith.submit": "Алға",
 
   // "browse.startsWith.type_date": "Filter results by date",
-  // TODO Source message changed - Revise the translation
   "browse.startsWith.type_date": "Немесе күнді енгізіңіз (жыл-ай):",
 
   // "browse.startsWith.type_date.label": "Or type in a date (year-month) and click on the Browse button",
-  // TODO New key - Add a translation
-  "browse.startsWith.type_date.label": "Or type in a date (year-month) and click on the Browse button",
+  "browse.startsWith.type_date.label": "Немесе күнді (жыл-ай) енгізіп, "шолу"түймесін басыңыз.",
```

Comment:
This line is causing an error when you try to build the UI using `yarn build:prod` (this is production mode build).  It looks like the problem is that you have nested quotes inside the value.  Those nested quotes need to be escaped by using `\"` instead of just `"`.